Source/WebCore/Modules/WebGPU/GPUCanvasContext.cpp

```diff
@@ -28,26 +28,144 @@
 
 #include "GPUAdapter.h"
 #include "GPUCanvasConfiguration.h"
+#include "RenderBox.h"
+#if PLATFORM(COCOA)
```

Comment:
Ifdef’d includes come after the other includes, with a blank line in between.

Reply:
Fixed.